src/main/kotlin/frc/robot/subsystems/conveyor/Conveyor.kt

```diff
@@ -9,12 +9,12 @@ import org.littletonrobotics.junction.AutoLogOutput
 import org.littletonrobotics.junction.Logger
 
 class Conveyor private constructor(private val io: ConveyorIO) : SubsystemBase() {
-    private val kP = LoggedTunableNumber("Conveyor/kP", ConveyorConstants.GAINS.kP)
-    private val kI = LoggedTunableNumber("Conveyor/kI", ConveyorConstants.GAINS.kI)
-    private val kD = LoggedTunableNumber("Conveyor/kD", ConveyorConstants.GAINS.kD)
-    private val kS = LoggedTunableNumber("Conveyor/kS", ConveyorConstants.GAINS.kS)
-    private val kV = LoggedTunableNumber("Conveyor/kV", ConveyorConstants.GAINS.kV)
-    private val kA = LoggedTunableNumber("Conveyor/kA", ConveyorConstants.GAINS.kA)
+    private val kP = LoggedTunableNumber("Conveyor/kP", GAINS.kP)
```

Comment:
Just a side note about tunable numbers: according to https://github.com/Mechanical-Advantage/AdvantageScope/issues/243 it seems that a new feature in AdvantageKit will "allow tuning values while using the AdvantageKit NT mode", so I wouldn't work on these and wait for the beta release.